metrics-exporter-prometheus/Cargo.toml

```diff
@@ -15,15 +15,21 @@ readme = "README.md"
 categories = ["development-tools::debugging"]
 keywords = ["metrics", "telemetry", "prometheus"]
 
+[features]
+default = ["tokio-exporter"]
+tokio-exporter = ["hyper", "tokio"]
+
 [dependencies]
-metrics = { version = "0.13.0-alpha.1", path = "../metrics" }
-metrics-util = { version = "0.4.0-alpha.1", path = "../metrics-util"}
+metrics = { version = "0.13.0-alpha.1", path="../metrics" }
+metrics-util = { version = "0.4.0-alpha.1", path="../metrics-util" }
```

Comment:
Did you mean to scrub the spaces around the equal sign here?

Reply:
Sorry circling back to this now, will fix this -> Did not mean to scrub them, I think this was a result of me testing my previous issue of the versions not seeming to be updated.